src/structs/Action.jl

```diff
@@ -44,17 +41,12 @@ It can be defined in many different ways.
     - **Default:** The default is `linear()`
     - Possible simple easing functions is `sineio()` for more check
         [Animations.jl](https://jkrumbiegel.github.io/Animations.jl/stable/)
-- func is either a `Function` or a `Transition`
-    - This is the actual action that is applied to the parent object.
-    - It can be either a general function which takes in the following four arguments
+- func is the function that describes the actual action
+    - It can be a general function which takes in the following four arguments
         - video, object, action, rel_frame
```

Comment:
Perhaps provide better explanation into what each of these four arguments are (e.g. where they come from, how they are used by the `Action`, etc.)

Reply:
Can this be done in the first tutorial? I think `rel_frame` can be explained but the other three are quite intuitive or not? One should have defined a `Video` an `object` and an `action` at this point already.

Reply:
That's true. I will add this to issue #254 so we don't forget the point made here.